ROADMAP.md

```diff
@@ -126,7 +126,7 @@
 | [`arrow-parens`]                    | 🌟  | [`arrow-parens`](https://eslint.org/docs/rules/arrow-parens)                                                                                                                                                                                                    |
 | [`arrow-return-shorthand`]          | 🌟  | [`arrow-body-style`](https://eslint.org/docs/rules/arrow-body-style)                                                                                                                                                                                            |
 | [`binary-expression-operand-order`] | 🌟  | [`yoda`](https://eslint.org/docs/rules/yoda)                                                                                                                                                                                                                    |
-| [`callable-types`]                  | 🛑  | N/A                                                                                                                                                                                                                                                             |
+| [`callable-types`]                  | ✅  | [`typescript/callable-types`]                                                                                                                                                                                                                                   |
```

Comment:
Don’t forget to add this reference to the section at the bottom of the page so the link works properly.